python/nav/ipdevpoll/plugins/cdp.py

```diff
@@ -40,7 +42,7 @@ class CDP(Plugin):
     device.
 
     """
-    cache = None
+    neighbors = None
     neighbors = None
```

Comment:
Uhm? Duplicate?

Reply:
Well, actually this was an automatic refactoring. The naming duplication was not intentional, so this likely broke something. This is why we have reviews!! :-)

Reply:
Turns out, the old `neighbors` variable was never used anywhere (it was set once in the `_process_cache` method, but never read by any code), so the simplest solution was to just remove it entirely.